src/IntegrationTests/IntegrationTests/OpenTelemetryTraceExporterSmokeTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Newtonsoft.Json;
+using IntegrationTests.Fixtures;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace IntegrationTests
+{
+    public class OpenTelemetryTraceExporterSmokeTest : IClassFixture<OpenTelemetryUsageApplicationFixture>
+    {
+        private readonly OpenTelemetryUsageApplicationFixture _fixture;
+
+        private const string _traceApiKey = "{YOUR_TRACE_API_KEY}";
+
+        private const string _insightsQueryApiKey = "{YOUR_INSIGHT_QUERY_API_KEY}";
+
+        private const string _insightsQueryApiEndpoint = "https://insights-api.newrelic.com";
+
+        private const string _traceEndPointUrl = "https://trace-api.newrelic.com/trace/v1";
+
+        private const string _accountNumber = "{YOUR_ACCOUNT_NUMBER}";
+
+        public OpenTelemetryTraceExporterSmokeTest(OpenTelemetryUsageApplicationFixture fixture, ITestOutputHelper output)
+        {
+            _fixture = fixture;
+            _fixture.TestLogger = output;
+
+            _fixture.Exercise = () =>
+            {
+                _fixture.MakeRequestToWeatherforecastEndpoint();
+            };
+
+            _fixture.SetEnvironmentVariables(new Dictionary<string, string>()
+            {
+                {"NewRelic:ApiKey", _traceApiKey},
+                {"NewRelic:TraceUrlOverride", _traceEndPointUrl}
+            });
+
+            _fixture.Initialize();
+
+            //Wait 10s for the data to show up on New Relic backend.
+            Thread.Sleep(10000);
+        }
+
+        [Fact(Skip = "Temporarily skipping this test so that the build pipeline won't fail. This is because this test requires setting Api keys manually.")]
```

Comment:
Is there a way we can store the secrets in azure pipeline and retrieve them here?
Otherwise, this seems like just a sample app.

Reply:
Yes, there is a way, and I deferred this work in another story in our backlog as we discussed. The first step is to get it run correctly on a local box, the next step is to have it run in azure pipeline.

Reply:
Ah, Right. I added that detail to the [story ](https://newrelic.atlassian.net/browse/DOTNET-4508)you created so I won't forget.